pkg/cvo/cvo.go

```diff
@@ -141,9 +141,9 @@ type Operator struct {
 	// via annotation
 	exclude string
 
-	// includeTechPreview is set to true when the CVO should create resources with the `release.openshift.io/feature-gate=TechPreviewNoUpgrade`
+	// reqiredFeatureSet is set to true when the CVO should create resources with the `release.openshift.io/feature-gate=TechPreviewNoUpgrade`
```

Comment:
nit: missing a u in `requiredFeatureSet` everywhere in this file

Reply:
and stepping back a bit further, can multiple feature sets be simultaneously enabled?  Maybe this should be `enabledFeatureSets []string`?

Reply:
Only one at a time enforced via the API type.   https://github.com/openshift/api/blob/master/config/v1/types_feature.go#L55